task/redhat-dependency-analytics/0.2/redhat-dependency-analytics.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
I wonder if we should start using `tekton.dev/v1`

Reply:
I think we should start using v1, and also change the minimum pipeline version then

Reply:
It would be much appreciated if this implementation could be postponed to the next release due to some time consuming factors